tests/foreman/cli/test_remoteexecution.py

```diff
@@ -583,6 +584,58 @@ def test_positive_run_reccuring_job(self, fixture_vmsetup, module_org):
         assert rec_logic['state'] == 'finished'
         assert rec_logic['iteration'] == '2'
 
+    @pytest.mark.tier3
+    @pytest.mark.parametrize('fixture_vmsetup', ['rhel7'], indirect=True)
+    def test_positive_run_concurrent_jobs(self, fixture_vmsetup, module_org):
+        """Tests Ansible REX concurent jobs without batch trigger
+
+        :id: ad0f108c-03f2-49c7-8732-b1056570567b
+
+        :Steps:
+
+            0. Create 2 hosts, disable foreman_tasks_proxy_batch_trigger
+
+            1. Run Ansible Command job with concurrency-setting
+
+        :expectedresults: multiple asserts along the code
+
+        :CaseAutomation: Automated
+
+        :CaseLevel: System
+
+        :BZ: 1817320
+
+        :parametrized: yes
+        """
+        param_name = 'foreman_tasks_proxy_batch_trigger'
+        GlobalParameter().set({'name': param_name, 'value': False})
+        self.org = module_org
+        self.client = fixture_vmsetup
```

Comment:
why bother storing these as class attributes?

Reply:
That line occurs multiple times in each class in the file, and `fixture_vmsetup` is function class, so the VM is just used per test.  Perhaps another PR to git rid of all use of `self` and change them all to just be client1?

Reply:
and at the same time check bugs cited in this file have UUIDs:
https://bugzilla.redhat.com/show_bug.cgi?id=1817320
https://bugzilla.redhat.com/show_bug.cgi?id=1451675
https://bugzilla.redhat.com/show_bug.cgi?id=1818076

Reply:
It's true these are no longer needed, removed the ones in this test, will adjust all others in separate PR. I added comments in the affected bzs